pkg/accounting/rollup/rollup.go

```diff
@@ -86,35 +106,71 @@ func (r *Rollup) RollupRaws(ctx context.Context) error {
 		if rollupStats[iDay][node] == nil {
 			rollupStats[iDay][node] = &accounting.Rollup{NodeID: node, StartTime: iDay}
 		}
-		//increment Rollups
+		//increment data at rest sum
 		switch tallyRow.DataType {
-		case accounting.BandwidthPut:
-			rollupStats[iDay][node].PutTotal += int64(tallyRow.DataTotal)
-		case accounting.BandwidthGet:
-			rollupStats[iDay][node].GetTotal += int64(tallyRow.DataTotal)
-		case accounting.BandwidthGetAudit:
-			rollupStats[iDay][node].GetAuditTotal += int64(tallyRow.DataTotal)
-		case accounting.BandwidthGetRepair:
-			rollupStats[iDay][node].GetRepairTotal += int64(tallyRow.DataTotal)
-		case accounting.BandwidthPutRepair:
-			rollupStats[iDay][node].PutRepairTotal += int64(tallyRow.DataTotal)
 		case accounting.AtRest:
 			rollupStats[iDay][node].AtRestTotal += tallyRow.DataTotal
 		default:
-			return Error.Wrap(fmt.Errorf("Bad tally datatype in Rollup : %d", tallyRow.DataType))
+			r.logger.Info("rollupStorage no longer supports non-accounting.AtRest datatypes")
 		}
 	}
 	//remove the latest day (which we cannot know is complete), then push to DB
 	latestTally = time.Date(latestTally.Year(), latestTally.Month(), latestTally.Day(), 0, 0, 0, 0, latestTally.Location())
 	delete(rollupStats, latestTally)
 	if len(rollupStats) == 0 {
 		r.logger.Info("Rollup only found tallies for today")
-		return nil
+		return time.Now(), nil
 	}
-	err = r.db.SaveRollup(ctx, latestTally, rollupStats)
+
+	return latestTally, Error.Wrap(r.db.DeleteRawBefore(ctx, latestTally))
```

Comment:
Looks like we'll need to return rollupStats since we're not saving it in this function anymore

Reply:
Also maybe we don't want to delete the raws until the end of `SaveRollup` just in case something happens before it's recorded